source/runtime_gui.cpp

```diff
@@ -285,6 +285,8 @@ void reshade::runtime::load_config_gui(const ini_file &config)
 	config.get("STYLE", "FPSScale", _fps_scale);
 	config.get("STYLE", "FrameRounding", imgui_style.FrameRounding);
 	config.get("STYLE", "GrabRounding", imgui_style.GrabRounding);
+	config.get("STYLE", "HdrOverlayBrightness", _hdr_overlay.brightness);
+	config.get("STYLE", "HdrOverlayOverwriteColorSpaceTo", _hdr_overlay.overwrite_color_space_to);
```

Comment:
I'm debating whether these are best kept in the `STYLE` or `OVERLAY` section ... Mmh.